lib/licensee/project_files/package_info.rb

```diff
@@ -8,7 +8,7 @@ def possible_matchers
         when '.json'
           [Matchers::NpmBower]
         else
-          if filename == 'DESCRIPTION' && content.start_with?('Package:')
+          if filename == 'DESCRIPTION' && content.include?('Package:')
```

Comment:
Does the line have to begin `Package?` (or can it be in the middle of a line, even if not the first)?